### ISSUE TITLE
possible python branch?

### ISSUE THREAD

**Issue by reteps** [aydenp/PowerSchool-API#2]
I would be willing to develop a python branch under this project if you could write the documentation. It would be nearly identical in every single function.

**Comment by aydenp** [comment]
This sounds good. Want to fork the project and start working on it?

**Comment by reteps** [comment]
sure. You are missing a couple branches on your end, maybe you could add:

```
assignments []assignmentScores combined w/ []assigments
attendance []attendance
attendanceCodes []attendanceCodes
assignmentCategories []assignmentCategories
finalGrades []finalGrades
```

**Comment by aydenp** [comment]
I wasn't able to add those because I am not able to test with an installation of PowerSchool with those available, but I've created it in an issue (#2)

**Comment by reteps** [comment]
i made it on mine (https://github.com/reteps/pywerschool)
